package.json

```diff
@@ -21,6 +21,7 @@
     "axios": "^1.6.0",
     "class-variance-authority": "^0.7.0",
     "lucide-react": "^0.277.0",
+    "nanoid": "^5.0.4",
```

Comment:
can we remove ^

Reply:
- we use it in form builder store to assign id when a new block is created.

Reply:
please delete this